src/components/RichTextEditor.tsx

```diff
@@ -20,28 +21,28 @@ const TipTap: React.FC = () => {
       TaskItem,
       Highlight,
     ],
-    content: `
-      <p>This isn’t bold.</p>
-      <p><strong>This is bold.</strong></p>
-      <p><b>And this.</b></p>
-      <p style="font-weight: bold">This as well.</p>
-      <p style="font-weight: bolder">Oh, and this!</p>
-      <p style="font-weight: 500">Cool, isn’t it!?</p>
-      <p style="font-weight: 999">Up to font weight 999!!!</p>
-    `,
+    content: `Start writing`,
+    // triggered on every change
+ onUpdate: ({ editor }) => {
+  const json = editor.getJSON()
+  // send the content to an API here
+},
```

Comment:
❌ 
as for this, you should receive a method from the props called for example `onSubmit` and you will call it here and pass the content to it

```ts
type Props = { onSubmit: (content: string) => void };
const RichTextEditor: React.FC<Props> = ({ onSubmit }) => {
...

onUpdate: ({ editor }) => {
  const htmlContent = editor.getHTML()
  onSubmit(htmlContent);
},


```